internal/sync/sync.go

```diff
@@ -41,6 +41,12 @@ type Sync struct {
 	Client *entclient.Client
 }
 
+// See https://lxadm.com/Rsync_exit_codes#List_of_standard_rsync_exit_codes.
+const (
+	ERR_RSYNC_INCOMPAT = 2
+	ERR_RSYNC_STREAM   = 12
```

Comment:
Constants in go follow the same casing as other variables/functions. 

And I'd suggest naming these like `rsyncExitCodeIncompat`, since these aren't go errors.

Reply:
Gah, old C habit again. Thanks. I'll follow the names of the error vars they replaced in the old code